src/AccessToken.php

```diff
@@ -103,57 +101,129 @@ public function verify($token, array $options = [])
         $certsLocation = isset($options['certsLocation'])
             ? $options['certsLocation']
             : self::FEDERATED_SIGNON_CERT_URL;
-
-        unset($options['audience'], $options['certsLocation']);
+        $cacheKey = isset($options['cacheKey'])
+            ? $options['cacheKey']
+            : $this->getCacheKeyFromCertLocation($certsLocation);
 
         // Check signature against each available cert.
         // allow the loop to complete unless a known bad result is encountered.
```

Comment:
nit: this comment should move to where the loop happens now.

Reply:
Actually, there is no loop, because the Firebase JWT library supports an array of certs. So I'll remove the comment.